Correct spec documentation for GitHubSource (#811)
knative/docs: eventing/README.md

  [SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#secretkeyselector-v1-core)
  containing a GitHub secret token for configuring a GitHub webhook. One of this
  or `accessToken` must be set.

> - `serviceAccountName`: `string` The name of the ServiceAccount to run the

Reviewer: ```suggestion
- `serviceAccountName`: `string` The name of the ServiceAccount running as the
```

>   container as.

Reviewer: ```suggestion
  container.
```
